src/lib/routes/frontsite/Frontsite.js

```diff
@@ -23,7 +23,8 @@ export default class FrontSite extends Component {
   };
 
   render() {
-    const staticPagesRoutes = getStaticPagesRoutes();
+    const { config } = this.props;
+    const staticPagesRoutes = getStaticPagesRoutes(config.uiConfig.staticPages);
```

Comment:
not a blocker: the config can be deconstructed (probably it show ESLint warning for this line):

```
const {config: {uiConfig: {staticPages}} };
```